extensions/applicationinsights-debugplugin-js/rollup.config.js

```diff
@@ -96,8 +117,10 @@ const nodeUmdRollupConfigFactory = (isProduction) => {
 };
 
 export default [
-  browserRollupConfigFactory(true),
-  browserRollupConfigFactory(false),
+  browserRollupConfigFactory(true, version),
+  browserRollupConfigFactory(false, version),
+  browserRollupConfigFactory(true, verParts[0]),
```

Comment:
This will be v2 -- if we update the version to 3.x.x then this automatically becomes v3